### PR TITLE
Entry form and render Cards

### PR DESCRIPTION
**Pull request by CasparVelsen**
In this PR I added a form to the formPage and build a function to render cards from the forms inputs.

### CODE REVIEW

File: src/components/EntryForm.js [hunk 1]
```diff
@@ -41,9 +51,13 @@ export default function EntryForm({ onCreateCard }) {
   function handleSubmit(event) {
     event.preventDefault();
     onCreateCard({
-      Date,
+      date: date.split(',').map(name => name.trim()),
+      time: time.split(',').map(name => name.trim()),
```
Comment (MerleMak) [inline src/components/EntryForm.js]: Are you sure you need the "split by comma" method for date and time? usually a date would be typed in with only numbers and dots.
Unless you are asking for two different dates separated by commas

File: src/components/EntryForm.js [hunk 2]
```diff
@@ -51,8 +51,8 @@ export default function EntryForm({ onCreateCard }) {
   function handleSubmit(event) {
     event.preventDefault();
     onCreateCard({
-      date: date.split(',').map(name => name.trim()),
-      time: time.split(',').map(name => name.trim()),
+      date,
+      time,
```
Comment (MerleMak) [inline src/components/EntryForm.js:141]: ok you already changed it here haha

File: src/components/EntryForm.js [hunk 3]
```diff
@@ -3,23 +3,58 @@ import styled from 'styled-components';
...
-          <div>
+          <Legend>general infos</Legend>
+          <Div>
             <label htmlFor="date">Date</label>
-            <input id="date" name="date" type="date" required />
-          </div>
-          <div>
+            <Input id="date" name="date" type="date" required />
+          </Div>
+          <Div>
```
[the remark this answers is not in the document]
Comment (CasparVelsen) [reply src/components/EntryForm.js:40]: Yeah good idea. I'll check that. Thanks

File: src/components/EntryForm.js [hunk 4]
```diff
@@ -1,14 +1,23 @@
 import styled from 'styled-components';
+import { useState } from 'react';
+
+export default function EntryForm({ onCreateCard }) {
+  const { Date, setDate } = useState('');
```
[the remark this answers is not in the document]
Comment (CasparVelsen) [reply src/components/EntryForm.js]: No, you're right! Thanks. I've got to change that

File: src/components/EntryForm.js (hunk 1 above)
Comment (CasparVelsen) [reply src/components/EntryForm.js]: I think I did that to test something. I can take them out by now. Good eyes 🕵️‍♂️

File: src/components/EntryForm.js [hunk 5]
```diff
@@ -64,14 +64,15 @@ export default function EntryForm({ onCreateCard }) {
...
-            <label htmlFor="water">Water</label>
+            <label htmlFor="water">water</label>
             <Input
               id="water"
               name="water"
               type="text"
               maxLength={25}
               onChange={event => setWater(event.target.value)}
               value={water}
+              placeholder="water"
```
[the remark this answers is not in the document]
Comment (CasparVelsen) [reply src/components/EntryForm.js:75]: haha I check that! Thanks alot

**Pull request merged**